client/src/components/StreamSelect.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import Input from "@material-ui/core/Input";
+import { withStyles } from '@material-ui/styles';
+
+const styles = {
+  mainInput: {
+    backgroundColor: "black",
+  },
+  innerInput: {
+    color: "white", 
+    margin: "0 10px 0 10px",
+    '&::placeholder': {
+      fontStyle: 'italic',
+    },
+  }
+}
+
+class StreamSelect extends React.Component {
+  constructor(props) {
+    super();
+    this.state = {
+      channel: "",
+    };
+  }
+
+  handleChange = (e) => {
+    this.setState({ channel: e.target.value });
+  }
+
+  keyPress = (e) => {
+    if (e.keyCode === 13) {
```

Comment:
It would be nice if this were a constant named ENTER_KEYCODE or similar. A comment would do as well.
- [ ] Add named constant or comment